Version 2.3.0 and newer not working due to missing dependencies

`$npm installl multi-ini`

```javascript
// index.js
const ini = require('multi-ini')
```

`node index.js`

```
node:internal/modules/cjs/loader:959
  throw err;
  ^

Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'
Require stack:
- /multi-ini-test/node_modules/multi-ini/lib/multi-ini-class.js
- /multi-ini-test/node_modules/multi-ini/lib/index.js
- /multi-ini-test/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/multi-ini-test/node_modules/multi-ini/lib/multi-ini-class.js:3:30)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Module.require (node:internal/modules/cjs/loader:1028:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/multi-ini-test/node_modules/multi-ini/lib/multi-ini-class.js',
    '/multi-ini-test/node_modules/multi-ini/lib/index.js',
    '/multi-ini-test/index.js'
  ]
}
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "multi-ini",
-  "version": "2.3.1",
+  "version": "2.3.2",
   "license": "MIT",
   "description": "An ini-file parser which supports multi line, multiple levels and arrays to get a maximum of compatibility with Zend config files.",
   "main": "lib/index.js",
@@ -52,7 +52,6 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.0.0",
     "@babel/register": "^7.0.0",
-    "@babel/runtime": "^7.0.0",
     "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
     "babel-loader": "^8.0.4",
@@ -67,6 +66,7 @@
     "sinon-chai": "^2.8.0"
   },
   "dependencies": {
+    "@babel/runtime": "^7.0.0",
     "lodash": "^4.0.0"
   }
 }

